src/data/fintype/basic.lean

```diff
@@ -1297,6 +1297,10 @@ by simp [fintype.subtype_card, finset.card_univ]
   (set.univ : set α).to_finset = finset.univ :=
 by { ext, simp only [set.mem_univ, mem_univ, set.mem_to_finset] }
 
+lemma set.card_to_finset_univ [fintype α] :
+  (set.univ : set α).to_finset.card = fintype.card α :=
```

Comment:
I've needed to use this seemingly missing lemma a number of times. Just flagging it to check that there's not another intended way to convert between `fintype` and `to_finset`.
https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there.20code.20for.20X.3F/topic/set.2Ecard_to_finset_univ/near/266439576